src/components/WebCLI/WebCLI.tsx

```diff
@@ -192,12 +213,16 @@ const WebCLI = ({
       setHistoryPosition(0);
     }
 
-    connection?.send(
-      JSON.stringify({
-        ...authentication,
-        commands: [command],
-      }),
-    );
+    if (connection.current?.isOpen()) {
+      connection.current?.send(
+        JSON.stringify({
+          ...authentication,
+          commands: [command],
+        }),
+      );
+    } else {
+      setInlineError(InlineErrors.CONNECTION, Label.NOT_OPEN_ERROR);
+    }
     sendAnalytics({
```

Comment:
Maybe this should be inside the `if (connection.current?.isOpen()) {` above?

Reply:
Oops :) Moved it inside the `if`. Thanks!